nodejs/week4/spam-detector/index.js

```diff
@@ -0,0 +1,89 @@
+class Email {
+	constructor(subject, body) {
+		this.subject = subject;
+		this.body = body;
+	}
+}
+
+class SpamDetector {
+	badWordCheck(email) {
+		const badWords = ["viagra", "offer", "free", "business proposal"];
+
+		const emailBody = email.body;
+		const emailBodyIntoArray = emailBody.split(" ");
+
+		const badWordsInEmail = emailBodyIntoArray.filter((item) => {
```

Comment:
Avoid vague names like item. 

```suggestion
		const badWordsInEmail = words.filter((word) => {
```

Reply:
okay